### PR TITLE
Removed the link that take user to list of bids submitted on a tender | Adniya

### PR DESCRIPTION
**Pull request by adniyaYousaf**

### CODE REVIEW

File: client/src/TenderList.js [hunk 1]
```diff
@@ -81,11 +81,7 @@ const TendersList = () => {
 								{tender.status}
 							</span>
 						</p>
-						<h2 className="title">
-							<a className="tender-id" href={`./bidding/${tender.id}`}>
-								{tender.title}
-							</a>
-						</h2>
+						<h2 className="title">{tender.title}</h2>
```
Comment (soonick) [inline client/src/TenderList.js:84]: Why did you remove the link?

Comment (adniyaYousaf) [reply client/src/TenderList.js:84]: Because only buyer will be able to see the bids not all users. I have that link in buyer tender list.
